src/components/board/card/Card.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import StyledCard from "./Card.styled";
+
+
+function Card() {
+	return (
+		<StyledCard>
+		</StyledCard>
```

Comment:
if the component has no the child you should use a closed jsx element <StyledCard />